cms/static/cms/js/modules/cms.base.js

```diff
@@ -39,6 +39,8 @@ var CMS = {
     $(document).ready(function () {
         CMS.API.Helpers = {
 
+            isTouch: window.ontouchstart !== undefined,
```

Comment:
having ontouchstart in window sadly doesn't really mean anything. 
we have the surface tablet that handles both mouse events and touch events, but the user might be hovering with mouse and we would still show the `tap here`.

can we instead detect if the sequence of events that started the showTooltip started with touchstart or mouseover?